specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/hyperdrive.json

```diff
@@ -0,0 +1,714 @@
+      "description": "Specifies the parameter expression in the format [min_value(float), max_value(float), q(int)]. Please refer https://docs.microsoft.com/en-us/python/api/azureml-train-core/azureml.train.hyperdrive.parameter_expressions?view=azure-ml-py#qloguniform-min-value--max-value--q- for more information.",
+      "example": [0.01, 0.05, 3]
+    },
+    "QNormalParameterExpression": {
+      "type": "array",
+      "items": {
+        "type": "number",
+        "format": "float"
+      },
+      "minItems": 3,
+      "maxItems": 3,
+      "description": "Specifies the parameter expression in the format [mu(float), sigma(float), q(int)]. Please refer https://docs.microsoft.com/en-us/python/api/azureml-train-core/azureml.train.hyperdrive.parameter_expressions?view=azure-ml-py#qnormal-mu--sigma--q- for more information.",
+      "example": [0.01, 0.05, 3]
+    },
+    "QUniformParameterExpression": {
+      "type": "array",
+      "items": {
+        "type": "number",
+        "format": "float"
+      },
+      "minItems": 3,
+      "maxItems": 3,
+      "description": "Specifies the parameter expression in the format [min_value(float), max_value(float), q(int)]. Please refer https://docs.microsoft.com/en-us/python/api/azureml-train-core/azureml.train.hyperdrive.parameter_expressions?view=azure-ml-py#quniform-min-value--max-value--q- for more information.",
+      "example": [0.01, 0.05, 3]
+    },
+    "RandIntParameterExpression": {
+      "type": "array",
+      "items": {
+        "type": "integer"
+      },
+      "minItems": 1,
+      "maxItems": 1,
+      "description": "Specifies the parameter expression in the format [upper]. Please refer https://docs.microsoft.com/en-us/python/api/azureml-train-core/azureml.train.hyperdrive.parameter_expressions?view=azure-ml-py#randint-upper- for more information.",
+      "example": [5]
+    },
+    "UniformParameterExpression": {
+      "type": "array",
+      "items": {
+        "type": "number",
+        "format": "float"
+      },
+      "minItems": 2,
+      "maxItems": 2,
+      "description": "Specifies the parameter expression in the format [min_value, max_value]. Please refer https://docs.microsoft.com/en-us/python/api/azureml-train-core/azureml.train.hyperdrive.parameter_expressions?view=azure-ml-py#uniform-min-value--max-value- for more information.",
+      "example": [0.01, 0.05]
+    },
+    "HyperDriveExperimentResponse": {
+      "description": "Response for HyperDrive_CreateExperiment in case of success.",
+      "properties": {
+        "code": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Response code."
+        },
+        "result": {
+          "description": "Details about created Hyperdrive run.",
+          "$ref": "#/definitions/HyperDriveExperiment"
+        }
+      },
+      "required": [
+        "code",
+        "result"
+      ],
+      "type": "object"
+    }
+  },
+  "parameters": {
+    "ArmScopeParameter": {
```

Comment:
Is this value supposed to be URL encoded?

Reply:
No.

Reply:
Then you must apply ["x-ms-skip-url-encoding": true](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-skip-url-encoding)

Reply:
Done.

Reply:
Fixed.